pages/desktop/frontend/details.py

```diff
@@ -690,12 +695,16 @@ def rating_login_button(self):
 
         @property
         def rating_stars(self):
-            # waits for the ratings stars to be fully loaded and editable
+            # waits for the rating stars to be loaded
             self.wait.until(
                 expected.invisibility_of_element_located(
                     self._loaded_rating_stars_locator
                 )
             )
+            # waits for the rating stars to be editable
+            self.wait.until(
+                expected.element_to_be_clickable(self._rating_stars_locator)
+            )
```

Comment:
Was this really necessary? Did this method, in its initial form, cause any test failures for you? I'm asking because I haven't noticed failures triggered by this method for a very long time.